### ISSUE TITLE
component not visible after install with joomla 3.8.3

### ISSUE THREAD

**Issue by advanm** [asikart/quickicons#48]
When installing quickicons the component won't show up when selecting this from the components menu.
it get a message HTTP 500 error That’s odd...  the website can’t display this page  The site may be under maintenance or could have a programming error.

all other installed components are working correctly

**Comment by asika32764** [comment]
Quickicons is an admin component, but the `Site maintenance` message seems like a front-end message. What position is the error page at?

**Comment by advanm** [comment]
![quickicons](https://user-images.githubusercontent.com/35521088/35033394-cd98050e-fb6a-11e7-8af7-a3654f964864.jpg)
i see this link in the URL https://www.fosterdoggies.eu/administrator/index.php?option=com_akquickicons and then i get the http 500 error, it worked in joomla 3.7 buit now it doesn't
but i see the standard icons created by the component but i can't add new ones or edit exisiting icons. when i  select the quicicons link in the components menu i get a http 500 error.

**Comment by advanm** [comment]
![joomla quickicons](https://user-images.githubusercontent.com/35521088/35038022-a3eba404-fb79-11e7-8b27-8b2e088f8940.jpg)

dit is what i get when i click on the link in components in de admin module

**Comment by advanm** [comment]
any news about this?
i tried to install earlier versions but al the same result. Als other components work Ok when i install new components. The icons added by default are visible but i'm unable to add new icons.
![quickicons1](https://user-images.githubusercontent.com/35521088/35089816-4ced5340-fc38-11e7-9c90-d5bcfd439099.jpg)

**Comment by advanm** [comment]
Hi,

I put on faulttracking system on my website and when i'm trying to open the link the following is diplayed

Whoops \ Exception \ ErrorException (E_COMPILE_ERROR) 
Cannot use Windwalker\Object\Object as Object because 'Object' is a special class name.
maybe this could be a clue for you what's going wrong.

**Comment by brianpeat** [comment]
I've got the same error. I don't think Windwalker is compatible with PHP 7, or at least 7.2 which is what I'm running now.

**Comment by advanm** [comment]
@brianpeat It was working for me with PHP 7.0 but i switched to php7.2 because the support for Joomla 3.8 stops at march 2017 with php7.0  Hopefully they gonna fix this in a later version. I really like this component.

**Comment by advanm** [comment]
@asika32764 is it possible to fix this for PHP 7.2 in a short term?
As far as i know this worked with php 7.0

**Comment by advanm** [comment]
Hi,
any news about this? is there coming a solution for using this component with php 7.2 ?

**Comment by cengavercelik** [comment]
Hi;
I am using it in a joomla project (version 3.8.10). Everything seems ok unless I change my php version. It works with php 7.0, but does not work with php 7.1 and 7.2.

I have to make it clear; the icons work in the control panel page in all php versions (7.0,7.1,7.2) but when I use php 7.1 or 7.2, I can't edit the icons since I can't access to the Asikart QuickIcons page from the components menu.

I am at the test phase of the project so it is not a problem for me now (I change the php version to 7.0 to edit the links and then go back to 7.2). But when I am done with the test phase and go live, I don't have a luxury to change the php version whenever when I like just to make changes for the icons.

So I also wonder if there is any attempt or plan to fix this. If so we want to be informed.

Best Regards!

**Comment by rmcdonald41** [comment]
Yes we are having the same PHP problem - is there any fix for this ?

**Comment by rmcdonald41** [comment]
> Hi;
> I am using it in a joomla project (version 3.8.10). Everything seems ok unless I change my php version. It works with php 7.0, but does not work with php 7.1 and 7.2.
> 
> I have to make it clear; the icons work in the control panel page in all php versions (7.0,7.1,7.2) but when I use php 7.1 or 7.2, I can't edit the icons since I can't access to the Asikart QuickIcons page from the components menu.
> 
> I am at the test phase of the project so it is not a problem for me now (I change the php version to 7.0 to edit the links and then go back to 7.2). But when I am done with the test phase and go live, I don't have a luxury to change the php version whenever when I like just to make changes for the icons.
> 
> So I also wonder if there is any attempt or plan to fix this. If so we want to be informed.
> 
> Best Regards!

yes use multiphp manager that does give you the luxury to change PHP simply

**Comment by wouter-galia** [comment]
Hi 

I have the same issue. Since the developer did not respond in a very long time I assume the component is no longer supported? 

This is very unfortunate, it was a unique extension. Did anyone find something similar?
